### PR TITLE
GPII-2179: Pin to a published version of infusion that includes deletion relaying and new transforms.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [https://issues.gpii.net/browse/GPII-2179](https://issues.gpii.net/browse/GPII-2179) for details.

### CODE REVIEW

**Comment by the-t-in-rtf** [comment]
@amb26, this should be reviewed once the infusion and docs PRs are merged.

File: package.json [hunk 1]
```diff
@@ -14,17 +14,17 @@
     "cookie-parser": "1.4.3",
     "express": "4.14.0",
     "express-session": "1.14.2",
-    "infusion": "2.0.0-dev.20161027T155641Z.0bcd507",
-    "resolve": "1.1.7"
+    "infusion": "2.0.0-dev.20161219T170555Z.5778f7e",
+    "resolve": "1.2.0"
```
Comment (amb26) [inline package.json]: As far as I can see, this dependency is not used within gpii-express. Also, if it were, the version you have nominated (and all currently released versions of "resolve") suffer from https://github.com/substack/node-resolve/issues/106 and should be replaced in the interim by "fluid-resolve".

File: package.json [hunk 2]
```diff
@@ -14,17 +14,17 @@
     "cookie-parser": "1.4.3",
     "express": "4.14.0",
     "express-session": "1.14.2",
-    "infusion": "2.0.0-dev.20161027T155641Z.0bcd507",
-    "resolve": "1.1.7"
+    "infusion": "2.0.0-dev.20161219T170555Z.5778f7e",
```
Comment (amb26) [inline package.json]: This version of Infusion has now been deprecated as a result of its faulty version number. I've just rolled 3.0.0-dev.20170127T130413Z.103de6e which includes your gpii-binder transforms, for which cheers.

File: package.json (hunk 1 above)
Comment (the-t-in-rtf) [inline package.json]: Good catch.  I reviewed in depth, the dev "request" dependency is also unused.  Both have been removed.

File: package.json (hunk 2 above)
Comment (the-t-in-rtf) [inline package.json]: Thanks, and done.

**Pull request merged**